### PR TITLE
[Tizen Studio] 4.1 release notes

### PR DESCRIPTION
**Pull request by manishtr**
Signed-off-by: manish.r <manish.r@samsung.com>

### Change Description ###

Tizen Studio 4.1 release notes, same as 4.0 release notes as there are no new feature addition

### CODE REVIEW

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@manishtr The release note seems to be quite similar to 4.0, the difference is just RT deprecation statement. 

Please incorporate the comment. 
Thanks

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - Tizen 6.0 platform support has been added.
+  - Battery Historian tool has been added to help analyze the battery usage.
+  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
+  - Toolchain has been updated to GCC 9.2 and LLVM 10.
+  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/4-1-release-notes.md:14]: The supported OS information is updated.

For now, [the related page](https://github.com/Samsung/tizen-docs/blob/master/docs/application/tizen-studio/setup/prerequisites.md#os-and-system-requirements) was updated at previous Tizen Studio 4.0 release.
Thus the page doesn't need update at this release.

From next time, if the supported OS information is changed, please cross check the prerequisites page for Tizen Studio also to match the OS support information.

**Review by mijin85cho — COMMENTED** [review]
Hello, @annie-samsung 
When you cross check this PR and merge this PR, there are some points you should check according to the contents.
Please don't forget below point when you do Tizen Studio release note review.

Please check this PR with @nawab-samsung and merge it when the review is finished.

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md (hunk 1 above)
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-1-release-notes.md:14]: OS Support is still the same, there is no change.

**Review by dkson95 — APPROVED** [review]
LGTM 👍

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md [hunk 2]
```diff
@@ -58,4 +58,3 @@
   - The Dynamic Analyzer cannot show lifecycle information for Web applications.
   - Widget applications cannot be profiled with the Dynamic Analyzer. They are hidden in the application list on the toolbar for all Tizen platforms, except Tizen 2.3.2.
   - The Dynamic Analyzer sometimes gets stuck which is caused by an internal security checking program.
-  - Right Click on Profiling window does not work in Mac OS.
```
Comment (dkson95) [inline docs/application/tizen-studio/release-notes/4-1-release-notes.md]: Okay, agreed!

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md (hunk 1 above)
Comment (nawab-samsung) [reply docs/application/tizen-studio/release-notes/4-1-release-notes.md:14]: @mijin85cho can I resolve this conversation?

**Review by nawab-samsung — APPROVED** [review]
Since this release note is mostly based on the previous release note. I have reviewed the changes and thus approving the PR

**Review by dkson95 — APPROVED** [review]
LGTM!

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md [hunk 3]
```diff
@@ -23,6 +23,7 @@
 - Tizen Studio
   - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
   - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
```
Comment (dkson95) [inline docs/application/tizen-studio/release-notes/4-1-release-notes.md:26]: @manishtr Only for native project : `tizen build -c gcc` has an issue with old CLI ?

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+### Deprecated
+   - Tizen RT has been deprecated from Tizen Studio 4.1.
+   - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+  
+### Known Issues
+
+- Tizen Studio
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
```
Comment (kvbhat) [inline docs/application/tizen-studio/release-notes/4-1-release-notes.md:26]: @manishtr This issue is only with macos Catalina and above. Can we capture the same here? This issue will not be present in Windows or MacOS.
We can also provide Guide to fix the same?

Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-1-release-notes.md]: applied, we will create a page in https://developer.tizen.org/community/tip-tech for it, thanks.

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md (hunk 1 above)
Comment (sangwook96) [reply docs/application/tizen-studio/release-notes/4-1-release-notes.md:14]: Could you change like that?
Please remove them
  - Tizen 6.0 platform support has been added.
  - Battery Historian tool has been added to help analyze the battery usage.
  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
  - Toolchain has been updated to GCC 9.2 and LLVM 10.
  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.

Please add them
- Update Tizen Mobile/Wearable 5.5 emulator image
- Update IoT Setup Manager

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- IDE
+  - Tizen 6.0 platform support has been added.
+  - Battery Historian tool has been added to help analyze the battery usage.
+  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
+  - Toolchain has been updated to GCC 9.2 and LLVM 10.
+  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.
+  - Support for aarch64 architecture on IoT 6.0 extension for RPi4 has been added.
+
+### Deprecated
+   - Tizen RT has been deprecated from Tizen Studio 4.1.
```
[the remark this answers is not in the document]
Comment (sangwook96) [reply docs/application/tizen-studio/release-notes/4-1-release-notes.md:15]: Please remove " - Support for AArch64 architecture on IoT 6.0 extension for RPi4 has been added."

**Comment by annie-samsung** [comment]
@manishtr, kindly update the release date as tomorrow in the release notes. Please check if there are any further changes.

**Comment by manishtr** [comment]
> @manishtr, kindly update the release date as tomorrow in the release notes. Please check if there are any further changes.

done, for now the date of 21 is looking confirmed.

File: docs/application/tizen-studio/release-notes/4-1-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+### New Features
+
+- IDE
+  - Tizen Mobile/Wearable 5.5 emulator image has been updated.
+  - IoT Setup Manager has been updated.
+
+
+### Deprecated
+   - Tizen RT has been deprecated from Tizen Studio 4.1.
```
Comment (annie-samsung) [inline docs/application/tizen-studio/release-notes/4-1-release-notes.md:15]: Should it read, 
  - Tizen RT IDE has been deprecated from Tizen Studio 4.1.
  
or

  - Tizen Studio for RT has been deprecated from Tizen Studio 4.1.

OR
 
-  Support for Tizen RT has been deprecated from Tizen Studio 4.1.

Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-1-release-notes.md]: done

**Review by annie-samsung — APPROVED** [review]
Approving

**Pull request merged**